internal/v5/api.go

```diff
@@ -1460,12 +1448,16 @@ func (h *ReqHandler) servePromulgate(id *router.ResolvedURL, w http.ResponseWrit
 	return nil
 }
 
-// validPublishChannels holds the set of channels that can
-// be the target of a publish request.
-var validPublishChannels = map[params.Channel]bool{
-	params.EdgeChannel:   true,
-	params.StableChannel: true,
-}
+// ValidChannels holds the set of all allowed channels
+// that can be passed as a "?channel=" parameter.
+var ValidChannels = func() map[params.Channel]bool {
```

Comment:
I think the distinction between this and charmstore.ValidChannels is a bit confusing. Can't we make do with only one ValidChannels value?